jobs/management/commands/populate_database.py

```diff
@@ -0,0 +1,152 @@
+        )
+        return company
+
+    def add_job(self, title, company, city, country, description):
+        job = Job.objects.get_or_create(
+            title=title,
+            company=company,
+            contact_email='example@example.com',
+            city=city,
+            country=country,
+            description=description,
+            ready_to_publish=True,
+        )
+        return job
+
+    def handle_noargs(self, **options):
+        # Adding some meetups with future dates.
+        self.add_meetup(
+            title='Django Girls Warsaw',
+            city='Warsaw',
+            country='PL',
+            description=read_lines(),
+            meetup_date='2015-04-01'
+        )
+        self.add_meetup(
+            title='Women in Technology',
+            city='London',
+            country='GB',
+            description=read_lines(),
+            meetup_date='2015-05-15'
+        )
+        self.add_meetup(
+            title='Learn javascript',
+            city='Paris',
+            country='FR',
+            description=read_lines(),
+            meetup_date='2015-06-12'
+        )
+
+        self.add_meetup(
+            title='Python breakfast',
+            city='Berlin',
+            country='DE',
+            description=read_lines(),
+            meetup_date='2015-07-01'
+        )
+
+        self.add_meetup(
+            title='Girls Meetup',
+            city='New York',
+            country='US',
+            description=read_lines(),
+            meetup_date='2015-08-01'
+        )
+
+        # Adding some companies.
+        self.add_company(
+            name='Google',
+            website='http://www.google.pl/about/careers/students/'
+        )
+
+        self.add_company(
+            name='Amazon',
+            website='http://www.amazon.jobs/team-category/university-recruiting'
+        )
+
+        self.add_company(
+            name='Digital Ocean',
+            website='https://careers.digitalocean.com/'
+        )
+
+        # Adding some job offers.
+        self.add_job(
+            title='Intern',
+            company=Company.objects.get(name='Google'),
+            city='London',
+            country='GB',
+            description=read_lines(),
+        )
+
+        self.add_job(
+            title='Software Development Engineer - Paid Internship',
+            company=Company.objects.get(name='Amazon'),
+            city='Gdansk',
+            country='PL',
+            description=read_lines(),
+        )
+
+        self.add_job(
+            title='Software Engineer, Front End',
+            company=Company.objects.get(name='Digital Ocean'),
+            city='New York',
+            country='US',
+            description=read_lines(),
+        )
+
+        def print_created_objects(model):
+            """Prints created objects on terminal."""
+            for created_object in model.objects.all():
+                self.stdout.write('-{0}'.format(created_object))
```

Comment:
Note that this prints ALL objects that existing in the database, and not just the ones created with the manage command (which is OK)